documentation/multiple-forms-of-artifact.md

```diff
@@ -60,20 +62,25 @@ import org.apache.commons.rdf.api.IRI;
  * Friend of a friend, v0.99
  */
 public class FOAF implements Vocab {
+    :
+    :
     /**
      * A person.
      *
      * Defined by the vocabulary: http://xmlns.com/foaf/0.1/
      */
     public static final IRI Person = valueFactory.createIRI(FULL_IRI("Person"));
+    :
+    :
 }
 ```
 
 ## Generating constants that provide access to term meta-data
 
-In many User Interface (UI) situations, it can be highly useful if the UI can
-be driven from meta-data described in the RDF vocabularies that describe the
-terms, concepts, and messages related to our specific problem domain.
+In many situations (particularly with User Interfaces (UI)), it can be very
+useful if the UI can be driven from meta-data described in the RDF
+vocabularies that we use to describe the terms, concepts, and messages related
+to our specific problem domain.
```

Comment:
this reads a little weirdly - what would be the situations _not_ involving UI when it would be useful if the UI can be driven from meta-data?

Reply:
Yeah, good point - I've reworked it: d542ec2